### PR TITLE
Date of Admission in Referral Form

### PR DESCRIPTION
**Pull request by peeyush14goyal**
Replaced consultation created at to admission date Issue #1157
Other files have changed due to Prettier

### CODE REVIEW

File: src/Components/Shifting/ShiftDetails.tsx [hunk 1]
```diff
@@ -418,7 +442,7 @@ export default function ShiftDetails(props: { id: string }) {
               <span className="font-semibold leading-relaxed">
                 Date of Admission:{" "}
               </span>
-              {moment(consultation.created_date).format("LL") || "-"}
+              {moment(consultation.admission_date).format("LL") || "-"}
```
Comment (gigincg) [inline src/Components/Shifting/ShiftDetails.tsx]: `admission_date` is optional on the db. Please use `created_date` when admission date is not present.

Comment (peeyush14goyal) [reply src/Components/Shifting/ShiftDetails.tsx]: @gigincg So I should do like this consultation.admission_date || consultation.created_date || "-" ?

Comment (bodhish) [reply src/Components/Shifting/ShiftDetails.tsx]: `consultation.admission_date || consultation.created_date`

Created date will be avilable in all scenarios

Comment (peeyush14goyal) [reply src/Components/Shifting/ShiftDetails.tsx]: Changes done @bodhish @gigincg. Review please

**Pull request merged**